run_service_ssh.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+if [ -z "$GGP_SDK_PATH" ]; then
+  echo "Ggp sdk not found"
+  exit 1
+fi
+
+GGP_EXEC="$GGP_SDK_PATH/dev/bin/ggp"
+
+if [ ! -z "$1" ]; then
+  eval $GGP_EXEC ssh put $1
+fi
+
+OUTPUT=$(eval $GGP_EXEC ssh init | tee /dev/tty)
+
+while IFS= read -r line; do
```

Comment:
I think
`$GGP_EXEC ssh init | tee /dev/tty | while read line; do`
is clearer.

Reply:
I agree, but I don't get it to work that way. If I understood it, piping to while creates a subshell with its own environment 

https://unix.stackexchange.com/questions/9954/why-is-my-variable-local-in-one-while-read-loop-but-not-in-another-seemingly

This is a solution mentioned there 
```
tee /dev/tty | {
  while read line; do
  ...
  done

  # use the variables set in the loop

  ssh -t ... 
}
```
This also does not work, because the ssh command then fails with:
```
Pseudo-terminal will not be allocated because stdin is not a terminal.
sudo: no tty present and no askpass program specified
```

If you have a solution, I am happy to hear about it. otherwise I leave it as is.

Reply:
Got it. Thanks!